source/interact-data/specify-query.txt

```diff
@@ -0,0 +1,476 @@
+   and field types, if the field being queried is defined in the model class.
+
+The examples in this section use the following model definition:
+
+.. code-block:: ruby
+
+   class Band
+     include Mongoid::Document
+ 
+     field :name, type: String
+     field :founded, type: Integer
+     field :m, as: :member_count, type: Integer
+ 
+     embeds_one :manager
+   end
+ 
+   class Manager
+     include Mongoid::Document
+ 
+     embedded_in :band
+ 
+     field :name, type: String
+   end
+
+Field Syntax
+~~~~~~~~~~~~
+
+The field querying syntax uses the basic {+language+} hashes. The keys
+can be symbols or strings and correspond to field names in MongoDB
+documents.
+
+The following code shows two equivalent queries that use field querying
+syntax to retrieve documents in which the ``name`` field value is
+``'Depeche Mode'``:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-simple-field-query
+   :end-before: end-simple-field-query
+   :language: ruby
+   :dedent:
+
+Query API Syntax
+~~~~~~~~~~~~~~~~
+
+You can specify a Query API operator on any field by using the hash
+syntax, as shown by the following equivalent queries:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-query-api-query
+   :end-before: end-query-api-query
+   :language: ruby
+   :dedent:
+
+Symbol Operator Syntax
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can specify Query API operators as methods on symbols for the
+respective field name, as shown in the following code:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-symbol-query
+   :end-before: end-symbol-query
+   :language: ruby
+   :dedent:
+
+Fields
+------
+
+This section describes how to perform queries on fields with different
+types of values.
+
+Defined Fields
+~~~~~~~~~~~~~~
+
+.. TODO add link to Fields page
+
+To query on a field, the field does not need to be in the
+the model class definition. However, if a field is defined in
+the model class, {+odm+} coerces query values to match the defined field
+types when constructing the query.
+
+The following code specifies a string value when querying on the
+``founded`` field. Because the ``founded`` field is defined in the model
+class to have ``Integer`` values, {+odm+} coerces the string ``'2020'``
+to ``2020`` when performing the query:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-defined-field-query
+   :end-before: end-defined-field-query
+   :language: ruby
+   :dedent:
+
+Raw Values
+~~~~~~~~~~
+
+To bypass {+odm+}'s query type coercion behavior and query
+directly for the raw-typed value in the database, wrap the query value in
+the ``Mongoid::RawValue`` class, as shown in the following code:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-raw-field-query
+   :end-before: end-raw-field-query
+   :language: ruby
+   :dedent:
+
+This behavior can be useful when working with legacy data.
```

Comment:
Nit: I don't think this sentence really adds anything. I would suggest removing it